system/Commands/Utilities/Optimize.php

```diff
@@ -52,15 +52,31 @@ final class Optimize extends BaseCommand
      *
      * @var string
      */
-    protected $usage = 'optimize';
+    protected $usage = 'optimize [-c] [-l] [-r]';
+
+    /**
+     * The Command's options
+     *
+     * @var array
+     */
+    protected $options = [
+        'c' => 'Enable config caching.',
+        'l' => 'Enable locator caching.',
+        'd' => 'Disable config and locator caching.',
+    ];
 
     /**
      * {@inheritDoc}
      */
     public function run(array $params)
     {
+        // Parse options
+        $enableConfigCache  = CLI::getOption('c');
+        $enableLocatorCache = CLI::getOption('l');
+        $restoreDefaults    = CLI::getOption('r');
```

Comment:
```suggestion
        $disable    = CLI::getOption('d');
```
and replace `$restoreDefaults` with `$disable`.